src/test/java/HW/HW3_1.java

```diff
@@ -0,0 +1,185 @@
+        By lastNameLocator = By.name(lastName);
+        WebElement lastNameElement = practice.getDriver().findElement(lastNameLocator);
+        lastNameElement.sendKeys("SecondName");
+
+        String number = "//input[@aria-label='Mobile number or email']";
+        By numberLocator = By.xpath(number);
+        WebElement numberElement = practice.getDriver().findElement(numberLocator);
+        numberElement.sendKeys("322322453");
+
+        String createPassword = "//input[@data-type='password']";
+        By createPasswordLocator = By.xpath(createPassword);
+        WebElement passwordElement = practice.getDriver().findElement(createPasswordLocator);
+        passwordElement.sendKeys("zum322!");
+
+        By monthLocator = By.id("month");
+        WebElement monthElement = practice.getDriver().findElement(monthLocator);
+        Select month = new Select(monthElement);
+        month.selectByVisibleText("Nov");
+
+        By dayLocator = By.name("birthday_day");
+        WebElement dayElement = practice.getDriver().findElement(dayLocator);
+        Select day = new Select(dayElement);
+        day.selectByValue("7");
+
+        By yearLocator = By.xpath("//select[@aria-label='Year']");
+        WebElement yearElement = practice.getDriver().findElement(yearLocator);
+        Select year = new Select(yearElement);
+        year.selectByVisibleText("1994");
+
+        String signInButtonValue = "websubmit";
+        By signInLocator_button = By.name(signInButtonValue);
+        WebElement signInButton = practice.getDriver().findElement(signInLocator_button);
+        signInButton.click();
+
+        practice.sleep(2);
+
+        String message = "//div[starts-with(text(),'Please choose a gender')]";
+        By messageLocator = By.xpath(message);
+        WebElement messageElement = practice.getDriver().findElement(messageLocator);
+        boolean messageDisp = messageElement.isDisplayed();
+        Assert.assertTrue(messageDisp, "Error");
+
+        practice.quitWebPage();
+    }
+
+    @Test
+    public void BlogPostverify() {
+
+
+        practice.openURL("https://www.darksky.net/");
+
+        String textPart = "Dark Sky API";
+        By textLocator = By.partialLinkText(textPart);
+        WebElement textElement = practice.getDriver().findElement(textLocator);
+        textElement.click();
+
+        practice.sleep(2);
+
+        String partText = "blog post";
+        By partLocator = By.partialLinkText(partText);
+        WebElement partElement = practice.getDriver().findElement(partLocator);
+        partElement.click();
+
+        String xpath_date = "//time[contains(text(),'June 7, 2021')]";
+        By dateLocator = By.xpath(xpath_date);
+        WebElement dateElement = practice.getDriver().findElement(dateLocator);
+        String date = dateElement.getText();
+
+
+        SimpleDateFormat day = new SimpleDateFormat("MMMM d, yyyy");
+
+        boolean isMatch = true;
+        try {
+            day.format(day.parse(date));
```

Comment:
Not sure what you wanna do here. Looks like you parsing `date` again formating the same thing; its like going back and forth.